### ISSUE TITLE
MSMS Plugin

### ISSUE THREAD

**Issue by keith923** [Electrostatics/APBS_Sphinx#30]
Create a plugin for http://mgl.scripps.edu/people/sanner/html/msms_home.html.  We'd have to decide if we want the user to plop the binary in the correct place, or if we want to automagically download it.

This will be a good plugin to have, if for no other reason than to have a plugin that exec's a pre-built binary (very loose integration with Sphinx).

**Comment by lwwilson1** [comment]
So is your idea here basically a wrapper around a system call? It's not a big executable, so I'm guessing if we include it in the software, we could just plop it down in some place where it always is in the same position relative to the plugin? We could do the same for NanoShaper as well. I also have some CMake stuff from legacy APBS that grabs the proper executable and plops it in your build based on your OS

**Comment by keith923** [comment]
Yes, that's the basic idea.  I'm not so sure about including the software.  It would depend on the licensing, and if we are allowed to distribute it.  If we went that route, then I suppose we'd have to include a binary for each OS, and then call the right one based on a runtime check.

Having CMake grab it is an interesting idea.  The only real problem with that is that we don't really have a top level build setup.  That's a problem for the Geometric Flow plugin as well.  I'm not really sure that I want to go that direction anyway.  My thinking now is that I'd like to be able to grab plugins sort of like pip does for Python packages.  But I don't want to invest time in that just yet.

Another possibility I suppose is to have that runtime OS check I mentioned above, and also have it grab the binary if it doesn't exist.  Right now I sort of like this idea the best.  It's a quick and simple hack, and we can make it more complicated in the future if necessary.
